target/linux/generic/hack-4.9/912-silence-xhci-td-warnings.patch

```diff
@@ -0,0 +1,33 @@
+From a6eac48b24096503358b6d4ab0d40deb004e5dfd Mon Sep 17 00:00:00 2001
+From: Daniel Engberg <daniel.engberg.lists@pyret.net>
```

Comment:
Why did you replace the original author? This patch wasn't written by you.

Keep the original commit/patch intact as much as possible.

Reply:
Calm down, as I stated on irc this is my first kernel patch submission and I did refer to the source if you read a few lines below. I'll change that to Oliver....